Adyen/Core/Components/Base/Component.swift

```diff
@@ -14,9 +14,14 @@ extension Component {
 
     /// Finalizes the payment if there is any, after being processed by payment provider.
     /// - Parameter success: The status of the payment.
-    /// :nodoc:
+    @available(*, deprecated, message: "This property will no longer be available.")
```

Comment:
can we specify which new API to use in the `message` here? same with other deprecations.
maybe something like `message: Use finalizeIfNeeded(with:, completion:) instead`